multisig/proposals/vip-002/vip-002-ethereum.ts

```diff
@@ -0,0 +1,503 @@
+      params: ["6171"],
+    },
+    {
+      target: POOL_REGISTRY,
+      signature: "addMarket((address,uint256,uint256,uint256,address,uint256,uint256))",
+      params: [
+        [
+          vcrvUSD_Core,
+          "780000000000000000",
+          "800000000000000000",
+          "10000000000000000000000",
+          TREASURY,
+          "50000000000000000000000000",
+          "45000000000000000000000000",
+        ],
+      ],
+    },
+    {
+      target: TREASURY,
+      signature: "withdrawTreasuryToken(address,uint256,address)",
+      params: [CRV, "20000000000000000000000", ETHEREUM_MULTISIG],
+    },
+    {
+      target: CRV,
+      signature: "approve(address,uint256)",
+      params: [POOL_REGISTRY, 0],
+    },
+    {
+      target: CRV,
+      signature: "approve(address,uint256)",
+      params: [POOL_REGISTRY, "20000000000000000000000"],
+    },
+    {
+      target: vCRV_Core,
+      signature: "setReduceReservesBlockDelta(uint256)",
+      params: ["6171"],
+    },
+    {
+      target: POOL_REGISTRY,
+      signature: "addMarket((address,uint256,uint256,uint256,address,uint256,uint256))",
+      params: [
+        [
+          vCRV_Core,
+          "300000000000000000",
+          "350000000000000000",
+          "20000000000000000000000",
+          TREASURY,
+          "2000000000000000000000000",
+          "1100000000000000000000000",
+        ],
+      ],
+    },
+    {
+      target: TREASURY,
+      signature: "withdrawTreasuryToken(address,uint256,address)",
+      params: [crvUSD, "10000000000000000000000", ETHEREUM_MULTISIG],
+    },
+    {
+      target: crvUSD,
+      signature: "approve(address,uint256)",
+      params: [POOL_REGISTRY, 0],
+    },
+    {
+      target: crvUSD,
+      signature: "approve(address,uint256)",
+      params: [POOL_REGISTRY, "10000000000000000000000"],
+    },
+    {
+      target: vcrvUSD_Curve,
+      signature: "setReduceReservesBlockDelta(uint256)",
+      params: ["6171"],
+    },
+    {
+      target: POOL_REGISTRY,
+      signature: "addMarket((address,uint256,uint256,uint256,address,uint256,uint256))",
+      params: [
+        [
+          vcrvUSD_Curve,
+          "450000000000000000",
+          "500000000000000000",
+          "10000000000000000000000",
+          TREASURY,
+          "2500000000000000000000000",
+          "2000000000000000000000000",
+        ],
+      ],
+    },
+    {
+      target: TREASURY,
+      signature: "withdrawTreasuryToken(address,uint256,address)",
+      params: [CRV, "20000000000000000000000", ETHEREUM_MULTISIG],
+    },
+    {
+      target: CRV,
+      signature: "approve(address,uint256)",
+      params: [POOL_REGISTRY, 0],
+    },
+    {
+      target: CRV,
+      signature: "approve(address,uint256)",
+      params: [POOL_REGISTRY, "20000000000000000000000"],
+    },
+    {
+      target: vCRV_Curve,
+      signature: "setReduceReservesBlockDelta(uint256)",
+      params: ["6171"],
```

Comment:
Assuming 12 seconds per block on Ethereum, in 24 hours there would be 7,200 blocks. The previous calculation was assuming 14 seconds per block.

```suggestion
      params: ["7200"],
```